nexus/src/app/background/driver.rs

```diff
@@ -179,6 +205,44 @@ impl Drop for Driver {
     }
 }
 
+/// Activates a background task
+///
+/// See [`crate::app::background`] module-level documentation for more on what
+/// that means.
+///
+/// Activators are created with [`Activator::new()`] and then wired up to
+/// specific background tasks using [`Driver::register()`].  If you call
+/// `Activator::activate()` before the activator is wired up to a background
+/// task, then once the Activator _is_ wired up to a task, that task will
+/// immediately be activated.
+///
+/// Activators are designed specifically so they can be created before the
+/// corresponding task has been created and then wired up with just an
+/// `&Activator` (not a `&mut Activator`).  See the [`super::init`] module-level
+/// documentation for more on why.
+pub struct Activator {
+    pub(super) notify: Arc<Notify>,
+    pub(super) wired_up: AtomicBool,
+}
+
+impl Activator {
+    /// Create an activator that is not yet wired up to any background task
+    pub fn new() -> Activator {
```

Comment:
Yeah.... it would definitely remove a bunch of boilerplate.  But it feels misleading to me to call this a "default" value.  That suggests to me that it's sort of a safe choice, a useful value, when it's not really.  This could be an argument for not calling the constructor `new()`, and actually at one point I called it `new_stub()` for exactly that reason, but it didn't seem clearer.  Maybe more consistent would be if I called the constructor `blank()` or something?